public/components/model_list/model_table.tsx

```diff
@@ -173,10 +182,65 @@ export function ModelTable(props: ModelTableProps) {
     <EuiBasicTable<ModelAggregateSearchItem>
       columns={columns}
       items={models}
-      pagination={pagination}
+      pagination={models.length > 0 ? pagination : undefined}
       onChange={handleChange}
       sorting={sorting}
       hasActions
+      noItemsMessage={
+        <div style={{ padding: '8px 0 12px 0' }}>
```

Comment:
Shall we `memo` what we passed to `noItemsMessage`? Otherwise it changes every time ModelTable rerender, then it loses the point of keeping `onChange` in `onChangeRef`